src/freenet/clients/http/N2NTMToadlet.java

```diff
@@ -269,6 +296,11 @@ public static void addUnsentMessageTextInfo(HTMLNode node, String message) {
 	public static void createN2NTMSendForm(boolean advancedMode,
 			HTMLNode contentNode, ToadletContext ctx, HashMap<String, String> peers)
 			throws ToadletContextClosedException, IOException {
+		createN2NTMSendForm(null, advancedMode, contentNode, ctx, peers, "");
+	}
+	public static void createN2NTMSendForm(HTMLNode pageNode, boolean advancedMode,
```

Comment:
And can I convince you to make this method private, because there’s no reason it shouldn’t be? 😁